helpers/mysql/lib/opentelemetry/helpers/mysql.rb

```diff
@@ -40,7 +40,7 @@ class MySQL
       # Ignore query names that might appear in comments prepended to the
       # statement.
       PREPENDED_COMMENTS_REGEX = %r{/\*(?:[^/]|/[^*])*?(?:\*/|/\*.*)}m
```

Comment:
```suggestion
      PREPENDED_COMMENTS_REGEX = %r{(?:\/\*.*?\*\/)}m
```
This matches the `multi_line_comments` update.